docs/index.md

```diff
@@ -41,21 +41,21 @@ See [forgehax](https://github.com/fr1kin/forgehax) for an equivalent. Some featu
 	
 ***
 
```

Comment:
Can we remove the `***\n`

Reply:
just that one or all of them??

Reply:
@edwr the highlighted one that's there

Reply:
Oh and the bottom one in images too please